python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -15,18 +15,27 @@
 #
 
 from bigdl.chronos.forecaster.abstract import Forecaster
-from bigdl.orca.data.shard import SparkXShards
 from bigdl.chronos.forecaster.utils import\
-    np_to_creator, set_pytorch_seed, check_data,\
-        xshard_to_np, np_to_xshard, loader_to_creator
-import numpy as np
-from bigdl.orca.learn.pytorch.estimator import Estimator
-from bigdl.orca.learn.metrics import MSE, MAE
+    np_to_creator, set_pytorch_seed, check_data, xshard_to_np, np_to_xshard
+
+from bigdl.nano.pytorch.trainer import Trainer
 
+import numpy as np
 import warnings
 import torch
+from functools import partial
+from torch.utils.data import TensorDataset, DataLoader
+from torchmetrics.functional import mean_squared_error, mean_absolute_error,\
+    mean_absolute_percentage_error, r2_score, symmetric_mean_absolute_percentage_error
 
-ORCA_METRICS = {"mse": MSE, "mae": MAE}
+TORCHMETRICS_REGRESSION_MAP = {
+    'mae': mean_absolute_error,
+    'mse': mean_squared_error,
+    'rmse': partial(mean_squared_error, squared=False),
+    'mape': mean_absolute_percentage_error,
+    'r2': r2_score,
+    'smape': symmetric_mean_absolute_percentage_error
+}
 
 
 class BasePytorchForecaster(Forecaster):
```

Comment:
base_forecaster only contains BasePytorchForcaster. maybe change the name to base_pytorch_forecaster to avoid confusing.

Reply:
Do you think seperating the models by frameworks make sense? This way we can write some imports in the pytorch/__ini__.py without worrying about importing too many extra modules. Users  can import models like below. 
```
import chronos.forecaster.pytorch.LSTMForecaster
````
instead of
```
import chronos.forecaster.lstm_forecaster.LSTMForecaster
```
Statistical and machine learning models can be put in another folder as forecaster.ml, or just leave it outside the folders.